ddtrace/telemetry/__init__.py

```diff
@@ -0,0 +1,37 @@
+import ddtrace.internal.telemetry.writer
```

Comment:
Rename this module from `ddtrace/telemetry/__init__.py` to `ddtrace/_telemetry.py`

This looks good, but we can keep this module private for now until we enable by default.